<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenerateHah</class>
 <widget class="QDialog" name="GenerateHah">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1350</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SmartImageComparison</string>
  </property>
  <widget class="QPushButton" name="Home">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmp">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>CompareImage</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>80</y>
     <width>231</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Upload image to generate Hash:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_image">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>136</y>
     <width>231</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="FileDialog">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>130</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Upload</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="FilePath">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>190</y>
     <width>511</width>
     <height>27</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="hashgen">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>430</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Generate</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="hash">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>470</y>
     <width>751</width>
     <height>27</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>190</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>390</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QComboBox" name="hashLength">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>390</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>16</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>32</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>64</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>128</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>256</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>360</y>
     <width>221</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Hash size from below</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>470</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <zorder>Home</zorder>
  <zorder>cmp</zorder>
  <zorder>label</zorder>
  <zorder>label_image</zorder>
  <zorder>FileDialog</zorder>
  <zorder>FilePath</zorder>
  <zorder>hashgen</zorder>
  <zorder>hash</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>hashLength</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
